aws/aws.go

```diff
@@ -166,6 +180,8 @@ func GetAllResources(regions []string, excludedRegions []string, excludeAfter ti
 
 // NukeAllResources - Nukes all aws resources
 func NukeAllResources(account *AwsAccountResources, regions []string) error {
+	// Resources more than 500 in number could cause AWS to throttle api calls
```

Comment:
Again, please mention that 500 is a rule of thumb based on experience, but the actual number will vary based on resource type, region, etc.